kubernetes/helm/pinot/templates/controller/service.yaml

```diff
@@ -31,5 +31,11 @@ spec:
     # [pod_name].[service_name].[namespace].svc.cluster.local
     - name: {{ .Values.controller.service.name }}
       port: {{ .Values.controller.service.port }}
+    {{- if .Values.controller.service.extraPorts }}
```

Comment:
Minor, just for consistency, add `if` to all the other services for extra ports. Thanks